index.html

```diff
@@ -77,54 +77,100 @@ <h2 class="text-heading">2</h2>
       <section id="amenities">
         <div class="background-tertiary px-5 pb-4">
           <h1 class="text-heading py-5">Amenities</h1>
-          <div class="card-content p-3">
-            <div class="row pb-3 ps-3">
-              <div class="col">
-                <i class="fa-solid fa-people-roof fa-2xl"></i>
-              </div>
-              <div class="col">
-                <i class="fa-solid fa-car fa-2xl"></i>
-              </div>
-              <div class="col">
-                <i class="fa-solid fa-warehouse fa-2xl"></i>
-              </div>
-              <div class="col">
-                <i class="fa-solid fa-dog fa-2xl"></i>
-              </div>
-              <div class="col">
-                <i class="fa-solid fa-person-swimming fa-2xl"></i>
+          <div class="row text-center justify-content-center">
```

Comment:
I looked up the documentation for the Bootstrap grid again and I saw the row column option. I think we should use that instead. It allows us to make changes without changing the classes of each child.

```suggestion
          <div class="row row-cols-2 row-cols-md-4 text-center justify-content-center">
```